encrypt.go

```diff
@@ -57,6 +57,9 @@ type encryptOptions struct {
 	OverrideDatasources string `long:"override-datasources" description:"Override the cloud-init datasources with the supplied comma-delimited list of sources"`
 	GrowRoot            bool   `long:"grow-root" description:"Grow the root partition to fill the available space, disabling cloud-init's cc_growpart"`
 
+	RootPartitionUUID string `long:"root-partition-uuid" description:"Explicitly provide the root partition UUID instead of relying on partition type"`
+	ESPPartitionUUID  string `long:"efi-partition-uuid"  description:"Explicitly override the ESP partition UUID instead of relying on partition type"`
```

Comment:
Redundant space before `description`

`Explicitly provide` instead of `Explicitly override` (for consistency with the line above)